assembl/static2/css/components/dateRangePicker.scss

```diff
@@ -0,0 +1,85 @@
+.date-range-picker {
+  margin-top: 10px;
+
+  .DateInput_input__small {
+    font-size: $font-size-sm;
+  }
+
+  .DateInput__small {
+    width: 156px;
+  }
+
+  .DateInput_input,
+  .DateInput_input__focused {
+    width: 68%;
+  }
+
+  input[aria-label='Start Date'] {
```

Comment:
If you are in French, the style won't be applied?

Reply:
No, this is an attribute coming from the library. It's always the same